lisa/sut_orchestrator/baremetal/cluster/cluster.py

```diff
@@ -11,6 +11,12 @@
 from ..schema import ClusterSchema
 
 
+class ClusterCapabilities:
```

Comment:
Do you want to get cluster/server capability, or client capability? For rack manager, it doesn't make sense to get cluster (rack manager) capability.

Reply:
I want to get client capability, to create node capabilities